api/client/login.go

```diff
@@ -26,7 +26,6 @@ func (cli *DockerCli) CmdLogin(args ...string) error {
 
 	flUser := cmd.String([]string{"u", "-username"}, "", "Username")
 	flPassword := cmd.String([]string{"p", "-password"}, "", "Password")
-	flEmail := cmd.String([]string{"e", "-email"}, "", "Email")
```

Comment:
@kencochrane yes, that's what I was thinking; reason for keeping the flag is that we may break scripts that people use, and they will end up with:

```
flag provided but not defined: --email
See 'docker run --help'.
```

I think it's better to keep the flag for that reason